### DISCUSSION TITLE
Email body translations

### DISCUSSION THREAD

**Discussion by Prestaworks** [PrestaShop/PrestaShop#35246] [Q&A]
So, we have this issue https://github.com/PrestaShop/PrestaShop/issues/34057 that has been around since forever. The TinyMCE corrects the html when translating the mail body causing the order_conf email to be broken.

However, there is now also a new way to handle email translations with the email templates.

Is this an issue that we can skip because the feature of changing email body in the normal translation will be removed in the future?

**Comment by kpodemski** [comment]
At some point, we have to move translations fully to the new email system. There's a legacy one and a modern one. I still haven't played too much with a modern one, but I noticed many email translation issues.

It's a complex subject, and what I recommend to people is that if they want to edit their emails, it's better to download an HTML and  TXT copy and make adjustments from some code editor :(

I honestly don't think that project members will have time for this subject in the near future. There are a few more critical subjects to tackle first.

**Reply by Prestaworks** [reply]
I think the new system works well for the emails, it's more difficult of course to change the layout, but that is supposed to happen with templates now and not translations so if the maintainers can confirm that the legacy system should be removed, maybe there will be a PR to fix that showing up on github.
